client/js/lounge.js

```diff
@@ -994,6 +1005,22 @@ $(function() {
 		});
 	});
 
+	windows.on("show", "#connect", function() {
+		var params = parseQuery();
+		// Possible parameters:  name, host, port, password, tls, nick, username, realname, join
+		for (var key in params) {
+			var value = params[key].toLowerCase();
```

Comment:
I think you need to do something like `.replace(/[^[a-z]/, '')` here to disallow possible abuse and escapes from the jquery selector.